rabbit-escape-engine/src/rabbitescape/engine/solution/NothingPlaying.java

```diff
@@ -0,0 +1,17 @@
+package rabbitescape.engine.solution;
+
+import rabbitescape.engine.World.CompletionState;
+
+final class NothingPlaying extends SolutionInterpreter
+{
+    public NothingPlaying()
+    {
+        super( SolutionParser.parse("1") );
+    }
+    
+    @Override
+    public SolutionTimeStep next( CompletionState worldState )
+    {
+        return new SolutionTimeStep( ++commandIndex, new TimeStepAction[] {} );
+    }
+}
```

Comment:
Or use the interface Iterator<Timestep> instead of taking in a SolutionInterpreter and make NothingPlaying just implement that. 

But I think you'll want SolutionInterpreter to keep emitting empty time steps here anyway (after the solution is finished).